tests/integration/client.test.ts

```diff
@@ -132,3 +135,30 @@ test('Client: merge handlers', async () => {
     expect(mock1).toBeCalledTimes(1)
     expect(mock2).toBeCalledTimes(1)
 })
+
+
+test('Client: ak/sk auth; request headers', async () => {
+    const ak = randomString(10)
+    const sk = randomString(20)
+    const configAkSk = cloudConfig('http://t-systems.com')
+        .withAKSK(ak, sk, 'eu-de')
+        .config
+    const clientAkSk = new Client(configAkSk)
+    const projectID = randomString(20)
+    const domainID = randomString(20)
+    fetchMock.mockOnce(async req => {
+        expect(req.url.endsWith('?name=eu-de')).toBeTruthy()
+        return `{"projects": [{ "id": "${projectID}", "domain_id": "${domainID}" }]}`
+    })
+    await clientAkSk.authenticate()
+    fetchMock.mockOnce(async req => {
+        expect(req.headers.get('x-sdk-date')).toBeTruthy()
+        expect(req.headers.get('x-domain-id')).toBe(domainID)
+        expect(req.headers.get('x-project-id')).toBe(projectID)
+        expect(req.headers.get('authorization')).toBeTruthy()
+        expect(req.headers.get('authorization')!.startsWith('SDK-HMAC-SHA256 Credential=')).toBeTruthy()
+        return ''
+    })
+    const iam = clientAkSk.getService(IdentityV3)
+    await iam.listProjects()
```

Comment:
should be here some `expects()`?

Reply:
They are in `fetchMock.mockOnce`, on the "server" side. `listProjects` here is used for sending any request